src/pages/rtl.tsx

```diff
@@ -0,0 +1,28 @@
+import {ThemeProvider, useThemeValue} from '@gravity-ui/uikit';
+import React, {useEffect} from 'react';
+
+import {Landing} from '../components/Landing/Landing';
+import {Layout} from '../components/Layout/Layout';
+
+export const RTLPage = () => {
+    const theme = useThemeValue();
+
+    // Workaround for missing direction 'ltr' in ThemeProvider
+    useEffect(() => {
+        document.getElementsByTagName('html')[0].setAttribute('dir', 'rtl');
+
```

Comment:
I wanna make issue to ThemeProvider contributors so that ltr direction is in provider